pom.xml

```diff
@@ -633,7 +633,14 @@
             <dependency>
                 <groupId>com.github.ben-manes.caffeine</groupId>
                 <artifactId>caffeine</artifactId>
-                <version>2.9.3</version>
+                <version>3.1.8</version>
```

Comment:
I think it would be good to register a ticket for upgrading this lib and link it here

Reply:
[WW-5393](https://issues.apache.org/jira/browse/WW-5393)